docs/scripts.js

```diff
@@ -4,7 +4,8 @@ document.addEventListener('DOMContentLoaded', function() {
     const hdmiFilterCheckbox = document.getElementById('hdmi-filter');
     const gridFilterCheckbox = document.getElementById('grid-filter');
     const languageFilterCheckbox = document.getElementById('language-filter');
-    const portfolioItems = document.querySelectorAll('.portfolio-item');
```

Comment:
Need access to the parent container now, so rather than query both up here figured I'd do `portfolioItemsContainer.children` below.